Cargo.toml

```diff
@@ -44,14 +44,13 @@ ibig = { version = "0.3", default-features = false, features = ["num-traits"] }
 impl-serde = { version = "0.4", default-features = false}
 lazy_static = "1"
 libsecp256k1 = { version = "0.7", default-features = false }
-near-crypto = "0.17"
+near-crypto = { git = "https://github.com/aleksuss/nearcore", rev = "6832608" }
```

Comment:
Here the fork is used because there is an [issue](https://github.com/near/nearcore/issues/10374) in the nearcore.